virtualization/windowscontainers/deploy-containers/containerd.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Building the container stack
+description: Learn more about new container building blocks available in Windows.
+keywords: LCOW, linux containers, docker, containers, containerd, cri, runhcs, runc
+author: scooley
+ms.date: 11/17/2018
+ms.topic: article
+ms.prod: windows-containers
+ms.service: windows-containers
+ms.assetid: a0e62b32-0c4c-4dd4-9956-8056e9abd9e5
+---
+
+# Container platform tools in Windows
+
+To make it easier for container management tools to run on a Windows with as little extra integration work as possible, we're introducing a Windows counterpart to runc - [runhcs](https://github.com/Microsoft/hcsshim/tree/master/cmd/runhcs).  We have also worked closely with the containerd project to integrate runhcs into [containerd/containerd](https://github.com/containerd/containerd) and [containerd/cri](https://github.com/containerd/cri).
```

Comment:
on a Windows -> on Windows ?

Reply:
Oh, I didn't notice the headline. I mean in the first line "to run on a Windows with" -> "to run on Windows with" ?